tests/factories/contacts/cht/clinic.js

```diff
@@ -0,0 +1,19 @@
+const Factory = require('rosie').Factory;
+
+const parent = { 
+  _id: 'hc1',
+  parent: {
+    _id: 'dist1'
+  } 
+};
+
+Factory.define('clinic')
+  .attr('_id','clinic1')
```

Comment:
If I call `Factory.build('clinic')` twice, what will the ids be?

```
const clinicOne = Factory.build('clinic');
const clinicTwo = Factory.build('clinic');

clinicOne._id === clinicTwo._id
```

I assume the value of both will be `'clinic1'`, which of course isn't what we're looking for. A better option would seem to be something like a generated UUID, which would better reflect "real" data and also avoid duplicates.

```
const uuidv4 = require('uuid/v4')

...

Factory.define('clinic')
  .attr('_id', () => { uuidv4() })  // their docs use this fn syntax, but I don't know why, where I suspect you can just pass `uuidv4` as the second arg to `attr`
```
There are times when we might not want a uuid, but that's part of the point of using something like Rosie here, since it's made to produce a bunch of defaults, but allow for overriding fields as needed.